locals.tf

```diff
@@ -169,6 +169,18 @@ locals {
             requests = { for k, v in var.resources.alertmanager.requests : k => v if v != null }
             limits   = { for k, v in var.resources.alertmanager.limits : k => v if v != null }
           }
+          storage = {
+            volumeClaimTemplate = {
+              spec = {
+                accessModes = ["ReadWriteOnce"]
+                resources = {
+                  requests = {
+                    storage = "2Gi"
```

Comment:
I would increase the size of this PVC to at least 10Gi. I know it's way over the required amount, but sadly, if we want to use Exoscale's block storage in the future, they don't allow volumes smaller than 10Gi.

If that's too big for you, then I would create a variable, set the default at 10Gi and you could then overload it on you side depending on the use cases.